services/connectivity/messaging/src/test/java/org/eclipse/ditto/services/connectivity/messaging/httppush/HttpPushClientActorTest.java

```diff
@@ -183,7 +183,7 @@ public void testTLSConnectionFails() {
                 .clientCertificate(TestConstants.Certificates.CLIENT_SELF_SIGNED_CRT)
                 .build();
         final SSLContext sslContext = SSLContextCreator.fromConnection(connection, DittoHeaders.empty())
-                .clientCertificate(credentials);
+                .get(credentials);
```

Comment:
Potential NPE in `ConnectionContext.https(SSLContext)` as `.get(credentials)` is `@Nullable`